chain/README-dev.md

```diff
@@ -6,13 +6,13 @@
    released
 1. Open and merge a PR `master` -> `laika-node/pre-release` (for
    testnet releases) or `master` -> `tlbc-node/pre-release` (for
-   mainnet releases) without waiting for a review. Make sure you have
-   called `bumpversion release` in the `chain/laika` or `chain/tlbc`
-   folders and the version contains the 'rc' modifier (release candidate).
-   This will built the docker image and push it to Docker Hub under
-   either `trustlines/tlbc-testnet-next:pre-release` (for testnet
-   releases) or `trustlines/tlbc-node-next:pre-release` (for mainnet
-   releases).
+   mainnet releases) without waiting for a review. Make sure to
```

Comment:
You need to call that BEFORE you open the PR, so now this is wrong.

Reply:
:heavy_check_mark: